tests/unit/MigrateTest.php

```diff
@@ -34,6 +35,19 @@ protected function _after()
 	 * @throws Exception
 	 */
 	public function testMigrate() {
+		$this->assertInstanceOf(MysqlConnection::class, $this->app->connection);
+		$this->assertTrue(method_exists($this->app->connection, 'getPdo'));
+		$this->assertInstanceOf(\PDO::class, $this->app->connection->pdo);
+		$migrationTable = $this->app->connection->migrationTable ?: 'migration';
+		if($this->app->connection->tableExists($migrationTable)) {
```

Comment:
Same here: one test path where condition is true and the other where it is false.

Reply:
Honestly, I don't go into testing inconsistent states of migrations and tables. If any similar case occurs, a migrate/reset is recommended (like here), and we start again with a clean slate. This is what we want to work and test.

Reply:
So maybe assert that `$this->app->connection->tableExists($migrationTable)` is `true` and remove `if` condition but keeps the code snippet it encapsulates...